client/app/pages/alert/AlertEdit.jsx

```diff
@@ -110,8 +112,8 @@ export default class AlertEdit extends React.Component {
             </Form>
             <div>
               <HelpTrigger className="f-13" type="ALERT_SETUP">
-                Setup Instructions <i className="fa fa-question-circle" aria-hidden="true" />
-                <span className="sr-only">(help)</span>
+                {i18next.t("Alerts:Setup Instructions")} <i className="fa fa-question-circle" aria-hidden="true" />
+                <span className="sr-only">{i18next.t("(help)")}</span>
```

Comment:
It's nice to leave non-translatable characters like `(` or `...` outside of the translation string. This will make it simpler for people to add translation since they won't be worry about the context where such texts are being used.

```suggestion
                <span className="sr-only">({i18next.t("Alerts:help")})</span>
```

Reply:
yes, (same reply as previous one :D ) I will remove the three dots and other characters from the translated text in a separate PR since these `Saving..` and `(Help)` for example are used across the app at many places... this is also why you find no namespace prefix to them since they are in the `common` default namespace